packer/media-service/provisioning.json

```diff
@@ -1,6 +1,6 @@
 {
     "variables": {
-        "machine_images_ami": "ami-6857e51b"
+        "machine_images_ami": "ami-161b9765"
```

Comment:
just confirming, this defines the new base ami to build the packer-generated local ami right? "**Am i right**?"